assets/dev/javascript/meta-box/line-chart.js

```diff
@@ -8,7 +8,7 @@ wps_js.render_line_chart = function (response, key) {
                 data: params['data'],
                 previousData: params['previousData']
             };
-            if(keyName !== 'wp-statistics-quickstats-widget'){
+            if(keyName !== 'wp_statistics_quickstats_metabox'){
                 wps_js.new_line_chart(data, `${keyName}-chart`);
             }else{
```

Comment:
Let's make sure the source is reformated and your editor loads the `.editorconfig` properly. @Nora-rad